mautic/mautic
# Campaign Details View tabbed content, allow filtering data by date range OR "to date" (current default)

![screen shot 2018-04-30 at 1 26 00 pm](https://user-images.githubusercontent.com/3016737/39440810-193fc4c8-4c7a-11e8-96bc-d7d63b8cbf09.png)
**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes
| Automated tests included? | No New tests
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Maintains default / current experience, but provides a way to filter tabbed data on Campaign Detail view by using the date range form. Updates data using ajax. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. 
2. 

#### Steps to test this PR:
1. Requires a campaign with decisions, actions and conditions configured and run cron / mautic console to trigger campaign runs
2. a) default content for tabs contain data for "Campaign To Date" as indicated by the toggle
    b) set new date range (no need to hit apply) and toggle a tab to "By Date Range" in either the Decision, Action or Condition tab - data should reload using ajax for each applicable tab
    c) set new date range using daterange form and click Apply on the daterange form - Page should reload and the toggle on the tab pages indicates "By Date Range"
    d) click "Campaign To Date" and data should update for each applicable tab using ajax.

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. Maintains default / current experience, but provides a way to filter tabbed data by using the date range form.
2.
